packages/core/src/lib/input/input.svelte

```diff
@@ -35,31 +35,20 @@ export let disabled = false;
 /** The state of the input (info, warn, error, success), if any. */
 export let state: InputState = 'none';
 
+/** The HTML input element. */
+export let input: HTMLInputElement = undefined!;
```

Comment:
😂 It's not as scary as it looks. Here we're just asserting that `input` will be defined by the time that it's used by the parent component. It's really just the explicit type definition of how binding is typed within components, and only necessary because it's a prop:

```svelte
<script lang='ts'>

let input: HTMLElement

onMount(() => {
  input.value = 3
})

</script>

<input bind:this={input} />
```

Reply:
Gotcha! Could you leave a comment explaining this? To the untrained eye it's definitely scary!